ui/html/mentorslist.html

```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
-                or just
-                help you to improve your IT skills and get ready for the IT business.
+                with a lot of awesome developers and I really want to help you to begin an IT.
             </p></br>
             <h4><button type="submit" onclick="document.getElementById('id02').style.display='block'"
-                style="width:auto;">apply</button></h4><h4><br>
+                style="width:auto;" && .main.display = "none">apply</button></h4><h4><br>
             <h5>by clicking <span class="black">"apply" </span> you accept and make a request that you want
                 to be a mentee of this specific mentor</h5><br>
         </div>
     </div>
     </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```

Comment:
i change this image with the which was un meaningful and i change it to image of a human